include/pcp/filter/pass_through_filter.hpp

```diff
@@ -0,0 +1,105 @@
+#ifndef PCP_FILTER_PASS_THROUGH_FILTER_HPP
+#define PCP_FILTER_PASS_THROUGH_FILTER_HPP
+
+/**
+ * @file
+ * @ingroup filter
+ */
+
+#include <algorithm>
+#include <pcp/kdtree/linked_kdtree.hpp>
+
+namespace pcp {
+
+namespace pass_through_filter {
+
+template <class ScalarType>
+struct construction_params_t
+{
+    using scalar_type             = ScalarType;
+    scalar_type filter_limit_min_ = std::numeric_limits<scalar_type>::lowest();
+    scalar_type filter_limit_max_ = std::numeric_limits<scalar_type>::max();
+    std::size_t index_to_filter_  = 0;
+};
+} // namespace pass_through_filter
+/**
+ * @brief Iterates through the entire input once, and for each point, will check
```

Comment:
I thought the pass through filter was for arbitrary predicates on the input point cloud. Maybe you can correct me on this.